starboard/nplb/BUILD.gn

```diff
@@ -265,8 +265,8 @@ target(gtest_target_type, "nplb") {
     cflags = [ "-Wno-enum-constexpr-conversion" ]
   }
 
-  #  TODO: b/297808555 - Add these tests for windows based platform modular builds.
-  if (sb_is_modular && !sb_is_evergreen && is_host_win) {
+  #  TODO: b/297808555 - Add these tests for modular builds.
+  if (sb_is_modular && !sb_is_evergreen) {
     sources -= [
       "maximum_player_configuration_explorer.cc",
```

Comment:
This also removes the tests from current modular linux builds which I would prefer not to do.

Reply:
Yeah, I know.
Will add a variable that platforms can turn on to escape those test.  I was hesitant to add a new variable just for that, but I think more test coverage would be better.